recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0-3" %}
+
+package:
+  name: r-gsw
```

Comment:
I don't like the package variable name. We rarely need to change it and variable, IMO, are for things  we change frequently. I understand that jinja variable for names help copy-n-paste to create a new recipe, but then again, why in conda-skeleton?

Last but not least, I usually download the tarball via the command line to check checksums, license files, create patches, etc. That URL with jinja variables is impossible to copy-n-paste making our life more difficult for no real advantage.

PS: I am +1 to put the version as jinja variable though because that needs frequent changes.